user_manual/files/access_webdav.rst

```diff
@@ -366,6 +366,28 @@ Known Problems
   Known problems and their solutions are documented in the KB2123563_ article.
 
 **Problem**
+  Cannot map ownCloud as a WebDAV drive in Windows using self-signed certificate.
+
+**Solution**
+  1) Go to the your ownCloud instance via your favorite web browser ,
```

Comment:
Thank you for writing this, @ujin-v! A few corrections: sentence should end in a period rather than a comma